### PR TITLE
Added /members/intro API

### PR DESCRIPTION
**Pull request by shubham-y**
Closes https://github.com/Real-Dev-Squad/website-backend/issues/472

Add on

- Added username while storing recruiter information
- Fixed member integration test for archiveMembers and moveTomembers

### CODE REVIEW

**Review by MehulKChaudhari — DISMISSED** [review]
LGTM :fire: Thanks for adding tests.

File: test/integration/recruiters.test.js [hunk 1]
```diff
@@ -69,4 +82,80 @@ describe("Recruiters", function () {
...
+              "company",
+              "first_name",
+              "last_name",
+              "designation",
+              "reason",
+              "email",
+              "currency",
+              "package",
+              "timestamp",
+              "package",
```
Comment (Pavangbhat) [inline test/integration/recruiters.test.js:130]: Looks like the second **package** is a mistake here?

**Review by Pavangbhat — COMMENTED** [review]
@shubham-y Please address this comment 👇 
https://github.com/Real-Dev-Squad/website-backend/pull/529/files#r806502592

File: test/integration/recruiters.test.js (hunk 1 above)
Comment (shubham-y) [reply test/integration/recruiters.test.js:130]: Thanks for pointing it out. I will fix that

**Review by Pavangbhat — APPROVED** [review]
Great work !!
Looks good to me

**Pull request merged**